image/preseed/reset.go

```diff
@@ -31,6 +31,50 @@ import (
 	apparmor_sandbox "github.com/snapcore/snapd/sandbox/apparmor"
 )
 
+// bash-completion symlinks; note there are symlinks that point at
+// completer, and symlinks that point at the completer symlinks.
+// e.g.
+// lxd.lxc -> /snap/core/current/usr/lib/snapd/complete.sh
+// lxc -> lxd.lxc
+func resetCompletionSymlinks(completersPath string, preseedChroot string) error {
+	files, err := ioutil.ReadDir(filepath.Join(preseedChroot, completersPath))
```

Comment:
nitpick: you are joining `preseedChroot` and `completersPath` below, too, so it would make sense to do it once here and remember the result in a variable.

Reply:
Done as @bboozzoo suggested.